module.json

```
@@ -2,17 +2,28 @@
   "name": "journal-links",
   "title": "Journal Links",
   "description": "Include links to entities that reference the viewed journal, actor, item, etc.",
-  "version": "0.3.1",
   "author": "dconley",
-  "esmodules": ["./scripts/index.js"],
-  "scripts": [],
-  "styles": [],
-  "packs": [],
-  "minimumCoreVersion": "0.6.6",
-  "compatibleCoreVersion": "0.6.6",
+  "authors": [],
   "url": "https://github.com/Sigafoos/journal-links",
   "license": "https://raw.githubusercontent.com/Sigafoos/journal-links/master/LICENSE",
   "bugs": "https://github.com/Sigafoos/journal-links/issues",
+  "flags": {},
+  "version": "0.4",
+  "minimumCoreVersion": "0.9",
+  "compatibleCoreVersion": "0.9",
```

```suggestion
  "compatibleCoreVersion": "9",
```
Needs to be set to 9 since Foundry changed their versioning